README.md

```diff
@@ -315,6 +317,19 @@ julia> filter(f -> endswith(f, ".log"), readdir(pwd()))
 
 The user implicitly controls when the files will be rolled over based on the `DateFormat` given.
 
+## `FormatLogger`
+The `FormatLogger` is a sink that formats the message and prints to a wrapped IO.
+Formatting is done by providing a function `f(io::IO, log_args::NamedTuple)`.
+
+```julia
+using Logging, LoggingExtras
+
+logger = FormatLogger() do io, args
+    printstyled(io, "[", args.level, "] "; color=Logging.default_logcolor(args.level))
+    println(io, args.message)
+end
+```
```

Comment:
maybe here include a actual example of the output,
and how it is different to the default output

Reply:
Sure, I originally had it but thought it was too long, but probably nice to show what is actually happening. I will choose another formatter too since this one is not so fun in a black-and-white readme and `Logging.default_logcolor` is not public interface IIUC.